test/integration/segmentation_camera_plugin.cc

```diff
@@ -97,22 +100,26 @@ void WaitForNewFrame()
 }
 
 /// \brief Build the scene with 3 boxes besides each other
-/// the 2 aside boxes have the same label & the middle is different
+/// the 2 outer boxes have the same label & the middle is different
 void BuildScene(rendering::ScenePtr scene)
 {
   math::Vector3d leftPosition(3, -1.5, 0);
   math::Vector3d rightPosition(3, 1.5, 0);
   math::Vector3d middlePosition(3, 0, 0);
+  math::Vector3d hiddenPosition(8, 0, 0);
```

Comment:
Can we add one more box that is partially hidden? Perhaps it can be poking out of the right of the left-most box. It'd be nice to add test coverage to make sure that for partially hidden objects, we only see the part that is visible in the segmented image

Reply:
It is already tested in the rendering::SegmentationCamera

Reply:
Hmm, it seems like there is some overlap/redundancy in testing between `ign-sensors` and `ign-rendering`, but I'm not sure what kind of tests should go in which repository for a given sensor and its rendering capabilities... @iche033 would you mind chiming in to help clarify which tests belong to which repository?